types/profiles.ts

```diff
@@ -1,5 +1,8 @@
-import * as light from "../build/ts/themes/light";
+import type * as light from "../build/ts/themes/light";
```

Comment:
Oooh yeah, this is real good 😎 Been seeing this error when we were doing our noodling